package.json

```diff
@@ -117,6 +120,7 @@
     "size-limit": "^4.5.0",
     "starwars-names": "^1.6.0",
     "url-loader": "^1.1.2",
+    "webpack": "^4.43.0",
```

Comment:
We're actually _only_ using `webpack` in the Storybook config file. Happy to remove this, but figured it's good to be explicit about dependencies.